vNext/test/NonSiloTests/project.json

```diff
@@ -1,5 +1,6 @@
-{
+﻿{
   "dependencies": {
+    "Google.Protobuf": "3.0.0",
```

Comment:
as we talked personally, but replying here in case someone was following, tests should pass in the crawl phase too. The difference is that tests will run in .NET Core too in the walk phase, while now we'll just run them in full .NET Framework referencing the .NET Standard binaries.